scripts.py

```diff
@@ -522,3 +523,26 @@ def do_run(self):
 
         print "\n".join(["%s %i (%s)" % l for l in sorted_languages])
         print json.dumps([l[0] for l in sorted_languages])
+
+class CompileTranslationsScript(Script):
+    """A script to combine translation files for circulation, core
+    and the admin interface, and compile the result to be used by the
+    app. The combination step is necessary because Flask-Babel does not
+    support multiple domains yet.
+    """
+
+    def run(self):
+        languages = Configuration.localization_languages()
```

Comment:
Since you do the three_to_two lookup at least, is it worth putting that code in localization_languages? The only reason I suggested using the 3-codes in the config file was for consistency. If every code point that uses then uses the 2-codes, it's fine to do the lookup immediately after reading from the config file.